hardening-check.yaml

```diff
@@ -0,0 +1,31 @@
+package:
+  name: hardening-check
```

Comment:
is the name specific enough?  i.e the repo is called devscripts, so wonder if `hardening-check-devscripts` or something else would be better?

Reply:
Other distros package this script stand alone in that package name. I.e. https://packagehub.suse.com/packages/hardening-check/ https://packages.fedoraproject.org/pkgs/hardening-wrapper/hardening-check/

Majority of other devscripts, are highly debian/ubuntu/.deb specific and have additional runtime dependencies on debian parsing libraries. Thus even if there is ever a need to package debian devscripts, this script likely will remain as a standalone package (or subpackage).

Reply:
got it - thanks